### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Local IIS Server_Keyboard: Focus is not visible on Add server button and More command button(...) of page.

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#59]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0), Chrome(68.0.3440.106)

Keyboard.
MAS ID Violated: MAS 2.4.7

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Local IIS link and Press Enter.
4.Tab till Add Server button and more command button and verify focus is visible on all controls of page.

Actual Result:
Focus is not visible on Add server button and More command button(...) of page.

Expected Result:
Keyboard focus should be clearly visible when focus is on any control of page.

User Impact:
Keyboard only users find difficulty in using the application if keyboard focus is not visible on any control

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B3b98b04f-4fab-4d3e-81af-a0906a34849c%7D

**Comment by Amlan454** [comment]
![bug_55_focus is not visible on add server and more command button](https://user-images.githubusercontent.com/42638683/44567086-74b4ca00-a78e-11e8-987d-5edcfb3756b7.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.4.7; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by fsteffi** [comment]
#A11ySev1;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
